packages/web-components/src/wallet-connection/makeAgoricSigner.ts

```diff
@@ -45,11 +30,14 @@ const zeroFee = () => {
 
 /**
  * Use a signing client to
```

Comment:
Looks like this comment was previously messed up. Should be something like "Use a signing client to create a signer for Agoric-specific transactions"